packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/__init__.py

```diff
@@ -23,26 +24,47 @@
 
 _instruments = ("openai >= 0.27.0",)
 
-WRAPPED_METHODS = [
+WRAPPED_METHODS_VERSION_0 = [
     {
+        "module": "openai",
         "object": "ChatCompletion",
         "method": "create",
         "span_name": "openai.chat",
     },
     {
+        "module": "openai",
         "object": "Completion",
         "method": "create",
         "span_name": "openai.completion",
     },
 ]
 
+WRAPPED_METHODS_VERSION_1 = [
+    {
+        "module": "openai.resources.chat.completions",
+        "object": "Completions",
+        "method": "create",
+        "span_name": "openai.chat",
+    },
+    {
+        "module": "openai.resources.completions",
+        "object": "Completions",
+        "method": "create",
+        "span_name": "openai.completion",
+    },
+]
+
```

Comment:
Consider preserving one variable, a dict though instead of an array where each key represents a major version and value an array of wrapped methods.

Might help when there will be another version, we'll be able to pick the right wrapped methods array by comparing the key with the sdk version

Reply:
Hope that they won't do any huge breaking changes in the future 😅